### PR TITLE
ci: Add CC Kubernetes job flags

### PR DESCRIPTION
**Pull request by Megan-Wright**
Adds flags for new K8s job types - `CC_SKOPEO_CRI_CONTAINERD_K8S` and `CC_CRI_CONTAINERD_K8S`

Fixes: #4557
Signed-off-by: Megan Wright <megan.wright@ibm.com>
Co-authored-by: Georgina Kinge <georgina.kinge@ibm.com>

### CODE REVIEW

File: .ci/ci_job_flags.sh [hunk 1]
```diff
@@ -96,21 +96,21 @@ case "${CI_JOB}" in
...
 			;;
-		"CC_CRI_CONTAINERD"|"CC_SKOPEO_CRI_CONTAINERD")
+		"CC_CRI_CONTAINERD"|"CC_SKOPEO_CRI_CONTAINERD" | "CC_CRI_CONTAINERD_K8S"|"CC_SKOPEO_CRI_CONTAINERD_K8S")
 			# Export any CC specific environment variables
 			export CCV0="yes"
 			export UMOCI=yes
-			if [ "${CI_JOB}" == "CC_SKOPEO_CRI_CONTAINERD" ]; then
+			if [ "${CI_JOB}" == "CC_SKOPEO_CRI_CONTAINERD" || "${CI_JOB}" == "CC_SKOPEO_CRI_CONTAINERD_K8S" ]; then
 				export SKOPEO=yes
 			fi
```
Comment (stevenhorsman) [inline .ci/ci_job_flags.sh:115]: You might want to consider changing
```
if [ "${CI_JOB}" == "CC_SKOPEO_CRI_CONTAINERD" || "${CI_JOB}" == "CC_SKOPEO_CRI_CONTAINERD_K8S" ]; then
	export SKOPEO=yes
fi
```
to
```
[[ "${CI_JOB}" =~ SKOPEO ]] && export SKOPEO=yes
```
as that matches the approach I've seen used elsewhere in the code base

File: .ci/run.sh [hunk 2]
```diff
@@ -61,7 +61,7 @@ case "${CI_JOB}" in
 		echo "INFO: Running tracing test"
 		sudo -E PATH="$PATH" bash -c "make tracing"
 		;;
-	"CC_CRI_CONTAINERD"|"CC_SKOPEO_CRI_CONTAINERD"|"CC_CRI_CONTAINERD_CLOUD_HYPERVISOR"|"CC_SKOPEO_CRI_CONTAINERD_CLOUD_HYPERVISOR")
+	"CC_CRI_CONTAINERD"|"CC_SKOPEO_CRI_CONTAINERD"|"CC_CRI_CONTAINERD_CLOUD_HYPERVISOR"|"CC_SKOPEO_CRI_CONTAINERD_CLOUD_HYPERVISOR"|"CC_CRI_CONTAINERD_K8S"|"CC_SKOPEO_CRI_CONTAINERD_K8S")
```
Comment (stevenhorsman) [inline .ci/run.sh:64]: I'm not sure you mean this? I think your new flags want to run something like:
```
echo "INFO: Running Confidential Container Kubernetes tests"
sudo -E PATH="$PATH" CRI_RUNTIME="containerd" bash -c "make cc-kubernetes"
```
?

Comment (Megan-Wright) [reply .ci/run.sh:64]: that makes more sense - would we then also need a change to `Makefile` to add `cc-kubernetes` as a command and point it to the correct tests?

Comment (stevenhorsman) [reply .ci/run.sh:64]: Yes, but I'd expect that change to be with your new test case changes (which could be part of this PR?)

Comment (wainersm) [reply .ci/run.sh:64]: yes, please, send the new tests alongside this changes so I can have it tested more easily :)

Comment (Megan-Wright) [reply .ci/run.sh:64]: ok that makes sense - in that case I'll close this PR as our test changes are on a different branch so it makes more sense to open a PR from there and add in these changes - thanks for the clarification!

**Comment by GeorginaKin** [comment]
Have opened a new PR here: https://github.com/kata-containers/tests/pull/4729

**Pull request closed without merging**